README.md

```diff
@@ -38,7 +38,7 @@ aws sts get-caller-identity --profile aws_profile
 
 ### Notes
 
-Okta account sessions are automatically cached when a global configuration file is present. As such, it is not recommended to run `bmx configure` or create a global configuration file on a machine with multiple users.
+Okta user sessions are automatically cached when a global configuration file is present. As such, it is not recommended to run `bmx configure` or create a global configuration file on a machine with multiple users.
```

Comment:
to further avoid confusion, let's use "account" for aws account exclusively, and "user" for Okta exclusively.